### PR TITLE
CLC-6195 - "Hub/CS/Crosswalk Status" dashboard, listing all CalCentral environments

### PR DESCRIPTION
**Pull request by winstonkim**
https://jira.ets.berkeley.edu/jira/browse/CLC-6195

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,16 @@
...
+npm start
+```
+And run the webpack (front end) server with:
+
+```bash
+webpack-dev-server
+```
+The web application can be accessed at [localhost:8080](http://localhost:8080/).
+## Screenshot
+![CalCentral Monitor Page Screenshot](http://s28.postimg.org/4u4govh7h/Screen_Shot_2016_04_07_at_4_58_21_PM.png)
```
Comment (christianvuerings) [inline README.md:16]: @winstonkim could we also add some comments about how `.env_template` and `.env` work?

File: src/client/components/app.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      </div>
+    );
+  }
+  componentDidMount() {
+    sockjs.onmessage = function(e) {
+      if (!e || !e.data) {
+        return;
+      }
+      var data = JSON.parse(e.data);
+      console.log('data:', data);
```
Comment (christianvuerings) [inline src/client/components/app.js]: @winstonkim could we remove the `console.log` for now? Or only show it when we're in debug mode?

File: src/client/components/config-item.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+          {this.generateEnvHeaders()}
+          {envItems}
+        </tbody>
+      </table>
+    );
+  }
+
+  generateEnvHeaders() {
+    var headers = [];
+    console.log(Object.keys(this.props.data.details)[0])
```
Comment (christianvuerings) [inline src/client/components/config-item.js]: @winstonkim same here, could we remove this for now?

File: src/client/components/app.js (hunk 2 above)
Comment (christianvuerings) [inline src/client/components/app.js]: @winstonkim perhaps add this rule to ESLint so we don't commit it

**Comment by winstonkim** [comment]
@christianvuerings, I've made the changes you requested. Please take a look!

**Comment by johncrossman** [comment]
@christianvuerings @winstonkim Perhaps we merge this _after_ the branch is cut today?

**Comment by christianvuerings** [comment]
@johncrossman FYI this is a separate repo atm so shouldn't impact the CalCentral one.

**Comment by johncrossman** [comment]
Got it. Thanks, @christianvuerings

**Pull request merged**